tests/mocha/server/sampleServerTest.js

```diff
@@ -0,0 +1,11 @@
+// Sample test generated by Velocity for reference
```

Comment:
For those lurking, Lars and I talked about this offline, and we agreed that it's okay to leave files like these in until we have written our own such tests.